packages/curvenote-cli/.eslintrc.cjs

```diff
@@ -1,4 +1,12 @@
 module.exports = {
   root: true,
   extends: ['curvenote'],
+  rules: {
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: ['**/*.spec.ts'],
+      },
+    ],
```

Comment:
This requires `vitest` to be in the dev deps of every package, which we don't want.

```
/* eslint import/no-extraneous-dependencies: ['off'] */
```

At the top of the spec file might be easier?

This doesn't trigger a lint warning anyways, just visually...